rrrspec-client/lib/rrrspec/client/cli.rb

```diff
@@ -89,6 +98,7 @@ def waitfor(taskset_id)
           elsif rs.size != 0
             rs[0].getc
             taskset.cancel
+            break
```

Comment:
Yes, it's the main purpose of this PR.

When `rrrspec waitfor` is interrupted, rrrspec is expected to cancel the taskset just like `rrrspec cancel` does.
`rrrspec cancel` doesn't care about the cancelled taskset.